test/unit/stats/service_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class Stats::ServiceTest < ActiveSupport::TestCase
   def setup
-    Timecop.return
+    travel_back
```

Comment:
now I see there rather need to go to `#teardown` to avoid keeping time modified for other tests

Reply:
In fact, it's not needed because all those tests call `travel_to` with a block, so the time is reset at the end. I removed it.